### PR TITLE
i18n(es): Translated `from-sveltekit.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `from-sveltekit.mdx` to spanish

### CODE REVIEW

File: src/content/docs/es/guides/migrate-to-astro/from-sveltekit.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Aunque el enrutamiento basado en archivos y los components de diseño son similares en Astro, es posible que desees leer acerca de la [estructura de proyectos de Astro](/es/core-concepts/project-structure/) para aprender dónde deben ubicarse los archivos.
+
+Para convertir otros tipos de sitios, como un portafolio o sitio de documentación, mira más plantillas oficiales de inicio en [astro.new](https://astro.new). Encontrarás un enlace al repositorio de GitHub de cada proyecto, así como enlaces de un solo clic para abrir un proyecto de trabajo en los entornos de desarrollo en línea de StackBlitz, CodeSandbox y Gitpod.
+
+## Recursos de la Comunidad
+
+- Artículo de blog: [Reescribiendo mi blog de SvelteKit a Astro](https://kharann.com/blog/rewriting-my-blog/)
+
+- ¡Añade el tuyo propio!
```
Comment (dreyfus92) [inline src/content/docs/es/guides/migrate-to-astro/from-sveltekit.mdx:73]:
```suggestion
- ¡Añade el tuyo!
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Gracias por la PR @Waxer59 , te dejo una sugerencia.

**Comment by Waxer59** [comment]
Gracias por la revisión @dreyfus92

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @Waxer59 & @dreyfus92 🙌

**Pull request merged**